src/hid-sensor.cpp

```diff
@@ -339,6 +351,34 @@ uint32_t hid_sensor::fps_to_sampling_frequency( rs2_stream stream, uint32_t fps
     else
         return fps;
 }
+void hid_sensor::set_imu_sensitivity( rs2_stream stream, float value ) 
+{
+    _imu_sensitivity_per_rs2_stream[stream] = value;
+}
+
+float hid_sensor::get_imu_sensitivity(rs2_stream stream)
+{
+    if( _imu_sensitivity_per_rs2_stream.find( stream ) != _imu_sensitivity_per_rs2_stream.end() )
+    {
+        return _imu_sensitivity_per_rs2_stream[stream];
+     }
+     else 
+        // gyro sensitivity default value is +-1000 therefore returning 30.5 resolution
+        // accel sensitivity default value is +-4g therefore returning 1.95 resolution
+        return stream == RS2_STREAM_GYRO ? 30.5f : 1.95f;
+}
+
+double hid_sensor::get_imu_sensitivity_converted( rs2_stream stream )
+{
+    if( _imu_sensitivity_per_rs2_stream.find( stream ) != _imu_sensitivity_per_rs2_stream.end() )
+    {
+        return gyro_sensitivity_convert.at( _imu_sensitivity_per_rs2_stream[stream] );
+    }
+    else
+        //gyro sensitivity default value is +-1000 therefore sending 0.1 
+        //accel sensitivity default value is +-4g therefore sensong 0.001
+        return stream == RS2_STREAM_GYRO ? 0.1f : 0.001f;
```

Comment:
Seems like we have 2 semilar function 1 in for converted and 1 is for not?

Before we had this comment

```
// gyro sensitivity default value is +-1000 therefore returning 30.5 resolution
        // accel sensitivity default value is +-4g therefore returning 1.95 resolution
```

+ typo `sensong`

Reply:
The first function - the converted function, is for values we send to FW the second function is for query the value for the viewer

Reply:
So maybe we call it `raw_values` and `values`?